docs/apikey.md

```diff
@@ -1,7 +1,19 @@
-# API Key
+# Create an API Key
 
-1. Click the user icon on the top-right corner.
-2. Choose Personal Settings
-3. Click the Create API Key button
-4. Set a description and choose an expiration date
-5. Save the API key securely in your computer. You are not going to retrieve the plain text key again in the console.
\ No newline at end of file
+You need to create an API key to access the Timeplus REST API. Here’s how to create one: 
+
+1. Click on your **avatar** in the top right corner. In the dropdown, click **Personal Settings**.
+
+<img width="1361" alt="api-key-avatar-1" src="https://github.com/timeplus-io/docs/assets/107260595/4aa9e9af-01e3-46e9-8724-aa10fde0da3e">
```

Comment:
should not use assets in this way. Either need to checkin the image or use a public available image.